os-partition/added/partitionPV.sh

```diff
@@ -34,9 +34,24 @@ function partitionPV() {
             # Not terminating, grab the lock without waiting
             flock -n $LOCK_FD
           else
-            # Terminating, grab the lock with timeout
-            echo "Existing server instance is terminating, waiting to acquire the lock"
-            flock -w $LOCK_TIMEOUT $LOCK_FD
+            REMAINING_LOCKING_TIME=$LOCK_TIMEOUT
+            echo "Existing server instance is terminating, attempting lock per second for $LOCK_TIMEOUT seconds"
+
+            while : ; do
+              flock -n $LOCK_FD
+              if [$? -eq 0 ]; then
```

Comment:
`if [ $? -eq 0 ]; then` -> missing space after [.

Reply:
@Crumby Thanks for catching that, addressed!